packages/ember-engines/addon-test-support/index.js

```diff
@@ -0,0 +1,60 @@
+import { getContext } from '@ember/test-helpers';
+
+/**
+ * Used to set up engine for use. Must be called after one of the `ember-qunit`
+ * `setup*Test()` methods.
+ *
+ *   Responsible for:
+ *     - create an engine object and set it on the provided context (e.g. `this.engine`)
+ *
+ * @method setupEngine
+ * @param {NestedHooks} hooks
+ * @param {String} engineName
+ * @public
+ */
+
+export async function setupEngine(hooks, engineName) {
+  hooks.beforeEach(async function() {
+    if (this.engine !== undefined) {
+      throw new Error('You cannot use `setupEngine` twice for the same test setup. If you need to setup multiple engines, use `loadEngine` directly.');
+    }
+
+    // setup `this.engine`
+    this.engine = await loadEngine(engineName);
+  });
+}
+
+function ownerHasEngine(owner, engineName) {
+  return Boolean(owner.factoryFor(`engine:${engineName}`));
+}
+
+async function buildEngineOwner(owner, engineName) {
+  let engineInstance;
+
+  if (ownerHasEngine(owner, engineName)) {
+    // eager engines
+    engineInstance = owner.buildChildEngineInstance(engineName, {
+      routable: false,
+      mountPoint: engineName,
+    });
+
+    await engineInstance.boot();
+  } else {
+    // lazy engines
+    engineInstance = await loadEngine(engineName);
+  }
+
+  return engineInstance;
+}
+
+async function loadEngine(engineName) {
+  let { owner } = getContext();
+  if (!ownerHasEngine(owner, engineName)) {
+    // ensure that the assets are fully loaded
+    let assetLoader = owner.lookup('service:asset-loader');
+
+    await assetLoader.loadBundle(name);
```

Comment:
@villander tests fail with `Assertion Failed: No bundle with name "" exists in the asset manifest.` error. I think this line should use `engineName` instead of `name`

Reply:
Thanks @SergeAstapov it's fixed. Now we need ember-qunit upgrade to get the CI green here.

Reply:
@villander it was done already

Reply:
It's not true @SergeAstapov the last `ember-qunit` is using the `@ember/test-helpers` 2.4.2, but I and @rwjblue r[eleased a 2.5.0](https://github.com/emberjs/ember-test-helpers/releases/tag/v2.5.0) toat enable ember-engines to run integration tests.

That said we need to merge this PR on `ember-qunit` repo and bum ember engines when it be released - https://github.com/emberjs/ember-qunit/pull/911

Reply:
@villander we have `"@ember/test-helpers": "^2.5.0"` listed in `ember-engines`, see https://github.com/ember-engines/ember-engines/blob/master/packages/ember-engines/package.json#L40

https://github.com/emberjs/ember-qunit/pull/911 is not needed - it's just a bump of `devDependency` in the `ember-qunit`.

Reply:
Ok, that's makes sense @SergeAstapov but I'm curious why it's not working 🤔

Reply:
@SergeAstapov the `ember-qunit` is responsible for exporting `setupRenderingTest`, not the test-helpers package 😄 

https://github.com/emberjs/ember-qunit/blob/472036008c90a8a504907626460a20601a8be82d/addon-test-support/index.js#L56